subprojects/logging/src/main/java/org/gradle/internal/logging/console/DefaultScalableElapsedTimeFormatter.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.logging.console;
+
+public class DefaultScalableElapsedTimeFormatter {
+    public String format(long elapsedTimeMs) {
+        double elapsedTime = elapsedTimeMs / 1000.0;
+
+        if (elapsedTime < 60.0) {
```

Comment:
Let's decide on the above before make any more changes. The only problem I had with removing tenth of seconds at the 10s mark that [it feel as if we hit a wall and everything feels slow moving forward](https://asciinema.org/a/4dopy383rq0b7j06icokc9zmj).